vassal-app/src/main/java/VASSAL/build/module/map/Flare.java

```diff
@@ -68,7 +68,7 @@
  */
 public class Flare extends AbstractConfigurable
         implements CommandEncoder, GameComponent, Drawable, MouseListener {
-  public static final String COMMAND_PREFIX = "FLARE:";
+  public static final String COMMAND_PREFIX = "FLARE\t";
```

Comment:
You can't just use arbitrary delimiters in the Command encoding unless you are really, really careful. The pattern is that Command parts are separated by Tabs.